cpp/example_code/s3/README.md

```diff
@@ -32,23 +39,31 @@ This is a workspace where you can find AWS SDK for C++ S3 examples utilizing the
 - [Uploading an object to an Amazon S3 bucket (asynchronously)](./put_object_async.cpp) (PutObjectAsync)
 - [Uploading an object to an Amazon S3 bucket (using a memory buffer instead of local disk copy)](./put_object_buffer.cpp) (PutObject)
 - [Configuring an Amazon S3 bucket for static website hosting](./put_website_config.cpp) (PutBucketWebsite)
-- [Finding, creating, and deleting an Amazon S3 bucket in a sequence](./s3-demo.cpp)
 
-## ⚠ Important
-- We recommend that you grant this code least privilege, or at most the minimum permissions required to perform the task. For more information, see [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) in the AWS Identity and Access Management User Guide.
-- This code has not been tested in all AWS Regions. Some AWS services are available only in specific [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
-- Running this code might result in charges to your AWS account. 
-- Running the unit tests might result in charges to your AWS account. [optional]
+### Scenarios
+- [Creating, listing, and deleting buckets](./s3_getting_started_scenario.cpp)
+- [Finding, creating, and deleting an Amazon S3 bucket in a sequence](./s3-demo.cpp)
 
-## Running the Examples
+## Running the examples
 Before using the code examples, first complete the installation and setup steps of [Getting Started](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started.html) in the AWS SDK for C++ Developer Guide.
 The Getting Started section covers how to obtain and build the SDK, and how to build your own code utilizing the SDK with a sample “Hello World”-style application. 
 
 Next, see [Getting started with the AWS SDK for C++ code examples](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started-code-examples.html) for information on the structure of the code examples, building, and running the examples.
 
-## Additional Information
-See [Amazon S3 code examples](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/examples-s3.html) in the AWS SDK for C++ Developer Guide for additional information on using the Amazon S3 service with the SDK.
+## Tests
+⚠️ Running the tests might result in charges to your AWS account.
+
+```sh
+   cd <BUILD_DIR>
+   cmake <path-to-root-of-this-source-code> -DBUILD_TESTS=ON
+   make
+   ctest
+   ctest --test-dir gtests 
+```   
 
-## Resources
+## Additional resources
+- [Amazon Simple Storage Service Documentation](https://docs.aws.amazon.com/s3/index.html)
+- [Amazon S3 code examples](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/examples-s3.html) in the AWS SDK for C++ Developer Guide for additional information on using the Amazon S3 service with the SDK.
```

Comment:
No further information required after this link (you can delete everything except the link):
- [Amazon S3 code examples](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/examples-s3.html)